doc-content/enterprise-only/integration/sso-central-client-create-proc.adoc

```diff
@@ -37,3 +37,13 @@ If you are configuring RH-SSO with {OPENSHIFT}, enter the URL that is exposed by
 After you create a new client, its *Access Type* is set to `public` by default. Change it to `confidential`.
 +
 The RH-SSO server is now configured with a realm with a client for {CENTRAL} applications and running and listening for HTTP connections at [parameter]``localhost:8180``. This realm provides different users, roles, and sessions for {CENTRAL} applications.
++
+[NOTE]
+====
+The RH-SSO server client uses one URL to a single {URL_COMPONENT_CENTRAL} deployment. The following error message may be displayed if there are two or more deployment configurations:
+"We are sorry... `Invalid parameter: redirect_uri`"
+To resolve this error, you must correct the *Valid Redirect URIs* field in the client configuration. On the *Configure* page, go to *Clients* > *kie* > *Settings*, and append the *Valid Redirect URIs* field with `/*`, for example:
```

Comment:
I would use something more specific than "correct", because you must specifically append to the content, while "correct" tends to mean "fix a mistake".

Not repeating comments for the other copies of this note.

Reply:
Changing "correct" to "append".